addons/assigngear/functions/fnc_gui_vehicleGear_load.sqf

```diff
@@ -0,0 +1,34 @@
+#include "\x\tmf\addons\assignGear\script_component.hpp"
+/*
+ * Name = TMF_assignGear_fnc_vehicleGear_save
+ * Author = Head
+ *
+ * Arguments:
+ * 0: ARRAY VehicleGear
+ *
+ * Return:
+ * Data to store
+ *
+ * Description:
+ * Internal Use Only
+ */
+params ['_value'];
+private _ctrlGroup = uiNamespace getVariable[QGVAR(vehicleGear_control), controlNull];
+private _categoryCtrl = _ctrlGroup controlsGroupCtrl IDC_VEHICLEGEAR_CATEGORY;
+private _factionCtrl = _ctrlGroup controlsGroupCtrl IDC_VEHICLEGEAR_FACTION;
+_value params [['_category', '', ['']], ['_faction', '', ['']], ['_cache', [ [], [], [] ], [], 3] ];
+TRACE_3('load',_category, _faction, _cache);
+GVAR(vehicleGear_data) = [_category, _faction, _cache];
+for [{ _i = 0 }, { _i < (lbSize  _categoryCtrl) }, { _i = _i + 1 }] do {
```

Comment:
Minor nitpick `for "_i" from 0 to ((lbSize _categoryCtrl) - 1) do {` is more efficient in SQF